md/single-sign-on-with-saml.md

```diff
@@ -247,22 +247,33 @@ com.bonitasoft.level = ALL
 
 ## Configure logout behaviour
 
-#### Bonita Portal
-
 When your Bonita platform is configured to manage authentication over SAML, when users log out of Bonita Portal, they do not log out of the SAML Identity Provider (IdP).  
 Therefore they are not logged out of all applications that are using the IdP.  
-To avoid this, you can hide the logout option of the portal.  
-To do this, set the `logout.link.hidden=true` option in `authenticationManager-config.properties` located in `platform_conf/initial/tenant_template_portal` 
-for not initialized platform or `platform_conf/current/tenant_template_portal` and `platform_conf/current/tenants/[TENANT_ID]/tenant_portal/`.
+To avoid this, you have two options :  
+
+#### Hide the logout button of the portal
+
+This is the most commonly used solution.  
```

Comment:
Without any "Logout" button, does this mean that Andy must close the PC session to validate the use case of logging out of all applications?

Reply:
At least close the web browser. But some SSO solutions work with the operating system session as well, so in that case, you need to close the OS session, indeed.
It depends on the SSO solution used. Usually that's with Kerberos on windows. Not sure such systems exist with SAML